docs/source/algorithms/ConvertPeaksWorkspace-v1.rst

```diff
@@ -0,0 +1,56 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm provides bi-directional conversion bewteen regular PeaksWorkspace and LeanElasticPeaksWorkspace.
```

Comment:
"bewteen" -> "between"

Reply:
the typo will be fixed in the next commit